src/index.html

```diff
@@ -1,13 +1,309 @@
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img
+              src="images/logo-2x.png"
+              alt="Base logo"
+              class="top-bar__logo"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">Bring joy back<br> to the everyday</h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+            src="images/logo-2x.png"
+            alt="Base logo"
+            class="top-bar__logo"
+            >
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#recommended">Products</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#categories">Store</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contact-us">Login</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">Cart(0)</a>
+              </li>
+            </ul>
+          </nav>
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+            +1 234 5555-55-55
+          </a>
+          <a href="tel:+1 234 5555-55-55" class="menu__call-to-order">
+            call to order
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <section class="recommended" id="recommended">
+            <h2 class="section-title">Recomended</h2>
+            <div class="recommended__products">
+              <article class="product recommended__product">
+                <img
+                  src="./images/products/Phone.png"
+                  alt="Nothing Phone"
+                  class="product__photo"
+                >
+                <h3 class="product__title">Phone (1)</h3>
+                <p class="product__description">Designed with instinct, to bring joy back to the everyday. Through the Glyph Interface, a perfected OS and exceptional dual camera. All startlingly fast.</p>
+                <p class="product__price">$ 499.00</p>
+              </article>
+
+              <article class="product recommended__product">
+                <img
```

Comment:
Code style here and on 90 line different for same element. Please, check code style entire document.